### ISSUE TITLE
Fix colors in search results page footer

### ISSUE THREAD

**Issue by atsansone** [flutter/website#10537]
From https://buganizer.corp.google.com/issues/337297674

### Observed Results

Having changed the CSS to forced colors, observe in the search results page that the  search button is not displayed properly also the footer.

* [Screenshot macOS](https://screenshot.googleplex.com/AWzbbqEtnjTTAZV.png)
* [Screenshot ChromeOS](https://screenshot.googleplex.com/6JToEPDEUo82Wdh.png))
* [Screenshot ChromeOS](https://screenshot.googleplex.com/EnNSnztvRyAPZwK.png)

### Expected Results

Ensure that the search button and the footer should be displayed with all styles and also the icon.

### User Impact

Low vision users are affected because the search button and the footer are not displayed accordingly.

### Steps to Reproduce

1. Enabled the CSS forced colors.
1. Open the following page: https://docs.flutter.dev/search?q=devtools
1. Continue the linear navigation using the <kbd>Tab</kbd> key.
1. Observe that the search icon has disappeared.
1. Also check the footer and the styles are disappeared

### GAR Assessment

[Flutter Docs](https://a11ytracker.corp.google.com/products/c2fdcf18-53d2-475d-b8b0-e5dedf8cb900?e=ProductLabels)

### GAR Details

[1.16. (d) With forced-colors (encountered by way of High Contrast Black on Windows)](go/GAR-web#run-through-forced-colors)

### Environment Details

Machine: MAC/ChromeBook
Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
Screen reader: NA.
Bug filling time: 15 mins.

**Comment by parlough** [comment]
These search results are generated, styled, and managed by the Google Custom Search setup we use.

Since we don't control that, I'm going to close this in favor of an umbrella issue https://github.com/flutter/website/issues/11800 which tracks implementing a new experience that we can properly integrate and style. Thanks!
